### DISCUSSION TITLE
Datasync.Client and MAUI

### DISCUSSION THREAD

**Discussion by ideas2appsaustralia** [Azure/azure-mobile-apps#349] [General]
Adrian

It appears that visual studio is already pushing MAUI. Is Microsoft.Datasync.Client expected to work with MAUI?

scott

**Comment by adrianhall** [comment]
I'm just working on a tutorial today.  Will let you know!

**Reply by adrianhall** [reply]
Follow-on - I've updated the library (5.0.3 has the MAUI support for offline).  Have not finished testing yet, so it's a WIP.  There is an issue associated with this.

**Reply by adrianhall** [reply]
And a further follow-on (and once I've done this, we can turn this into a MAUI discussion).

* New MAUI Sample published.
* New MAUI Tutorials going through editorial now.
* v5.0.3 seems to fix the MAUI problems, although it also creates issues for WinUI3
